modAionImpl/src/org/aion/zero/impl/sync/TaskGetStatus.java

```diff
@@ -71,19 +68,24 @@
     @Override
     public void run() {
         while (this.run.get()) {
-            Set<Integer> ids = new HashSet<>(p2p.getActiveNodes().keySet());
-
             try {
+                Set<Integer> ids = new HashSet<>(p2p.getActiveNodes().keySet());
+
                 for (int id : ids) {
                     p2p.send(id, reqStatus);
-                    Thread.sleep(1000L);
+                    Thread.sleep(interval / ids.size());
```

Comment:
Quick question, on the weekend we discussed a change of TaskGetStatus to:

~~~~java
for (int id : ids) {
  p2p.send(id, reqStatus);
}
Thread.sleep(<someTimeout>)
~~~~

Meaning status requests are sent out as a batch. Would this impact the algorithm in any discernable way?